### PR TITLE
5490 tweaks

### PR DESCRIPTION
**Pull request by cvalarida**
Connects to https://github.com/department-of-veterans-affairs/vets.gov-team/issues/1659  (Most of the items here have been done in https://github.com/department-of-veterans-affairs/vets-website/pull/5118)
Connects to https://github.com/department-of-veterans-affairs/vets.gov-team/issues/1769
Connects to https://github.com/department-of-veterans-affairs/vets.gov-team/issues/1792

Updates the advice given to "work with a trained professional" per https://github.com/department-of-veterans-affairs/vets.gov-team/issues/1659#issuecomment-288127827
image

Updates the 1990 intro page per https://github.com/department-of-veterans-affairs/vets.gov-team/issues/1769#issuecomment-288076695:
image

Updates the 5490 intro page per https://github.com/department-of-veterans-affairs/vets.gov-team/issues/1792
image

### CODE REVIEW

File: src/js/edu-benefits/5490/components/IntroductionPage.jsx [hunk 1]
```diff
@@ -34,7 +34,7 @@ class IntroductionPage extends React.Component {
                   <span>You won’t be able to save your work or come back to finish. So before you start, it’s a good idea to gather information about your education history and the school you want to attend.</span>
                 </div>
               </div>
-              <p>We strongly encourage you to work with a trained professional who can help you pick the right program. <a href="/disability-benefits/apply-for-benefits/help/index.html">Find an accredited representative.</a></p>
+              <p>We encourage you to work with a trained professional who can help you pick the right program. Call <a href="tel:888-442-4551">888-442-4551</a> (888-GI-BILL-1) from 8:00 a.m. to 7:00 p.m. (ET) to talk with an education specialist</p>
```
Comment (raquelromano-gov) [inline src/js/edu-benefits/5490/components/IntroductionPage.jsx]: Looks like "Tom" has yet to approve that they want only the phone number and not the link, but I say merge and we'll hear back if it's needed.

**Review by raquelromano-gov — COMMENTED** [review]
It looks like there are additional changes to the introduction page (e.g. edits to the list of stuff under "Prepare",  changes to the "You won’t be able to save your work" line, and the "What should I do while I wait? " stuff). Did you intend to add those here while you're updating it?

**Comment by cvalarida** [comment]
Yeah, sounds good. This PR was originally for https://github.com/department-of-veterans-affairs/vets.gov-team/issues/1659, but as I was going through the tickets, I found more for editing the intro page, so as I find them, I'll add them to this one.

**Comment by cvalarida** [comment]
@raquelromano-gov Where did you see those other edits to the intro page? I'm not finding them immediately...

**Comment by raquelromano-gov** [comment]
From your first issue above: #1659

**Comment by cvalarida** [comment]
Ohh, gotcha. Yeah, I already did most of that in https://github.com/department-of-veterans-affairs/vets-website/pull/5118, so I didn't duplicate it here. Should have mentioned that, though.

**Comment by raquelromano-gov** [comment]
Oh, OK! LGTM

**Review by raquelromano-gov — APPROVED** [review]

**Pull request merged**